.github/workflows/release.yml

```diff
@@ -65,19 +65,19 @@ jobs:
           cp "target/${{ matrix.build.TARGET }}/release/${binary_name}" "bin/${binary_name}-${{ matrix.build.NAME }}${extension}"
 
       - name: Upload binary
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: built-binaries
           path: bin/*
 
   release:
     name: Release
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Why don't we pin this to whatever the current `latest` points to. I'm assuming `ubuntu-24.04`.

Reply:
My rational was that this step does not really depend on anything special provided by the OS, putting this on an image with a moving target will reduce maintenance effort. But I'm glad to roll this back.

Reply:
Pinned all ubuntu jobs to 24.04